src/main/java/nextstep/qna/domain/BaseModel.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.qna.domain;
+
+import java.time.LocalDateTime;
+
+public abstract class BaseModel {
+
+    public LocalDateTime createdDate = LocalDateTime.now();
+
+    public LocalDateTime updatedDate;
+
+    public boolean deleted = false;
```

Comment:
```suggestion
    private LocalDateTime createdDate = LocalDateTime.now();

    private LocalDateTime updatedDate;

    private boolean deleted = false;
```
모든 인스턴스 변수는 가능하면 private으로 구현